backends/platform/androidsdl/androidsdl-sdl.h

```diff
@@ -37,10 +37,8 @@ class OSystem_ANDROIDSDL : public OSystem_POSIX {
 	void switchToRelativeMouseMode();
 	void showOnScreenControl(bool enable);
 
-#ifdef ENABLE_KEYMAPPER
 	// FIXME: This just calls parent methods, is it needed?
 	virtual Common::HardwareInputSet *getHardwareInputSet();
```

Comment:
I noticed that this is removed in PR #2018, however it would be a good idea to remove it here as well in case this PR is merged ahead of the others to avoid linker errors.